pkgs/applications/version-management/sr-ht/todo.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit
+, srht, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
```

Comment:
```suggestion
buildPythonApplication rec {
```

Reply:
Switch the all but `core-sr-ht` to `buildPythonApplication`, since it looks like `core-sr-ht` is suppose to be a python module, `srht`, I'm also guessing I should now move `core-sr-ht` into `pkgs/development/python-modules`?

Reply:
> I'm also guessing now I should move core-sr-ht into pkgs/development/python-modules?

If it's not going to be used by any other package, you might as well keep it here. But definitely remove it from `all-packages.nix`.